<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OTKControlWindow</class>
 <widget class="QDialog" name="OTKControlWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Проверка ОТК</string>
  </property>
  <widget class="QPushButton" name="pbOK">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>480</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>520</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>771</width>
     <height>591</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Изделия</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>353</y>
       <width>140</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Забракованные модули</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbDelCheckProd">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>355</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QListWidget" name="lwBrokenProd">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>380</y>
       <width>370</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Поиск по серийному номеру</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>353</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Проверенные модули</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Изделия для проверки</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leNumProduct">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>261</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbCheckProd">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>320</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Исправен</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbBrokenProd">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>320</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Неисправен</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbDelBrokenProd">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>354</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QListWidget" name="lwProduct">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>751</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="lwCheckedProd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>370</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Модули</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Поиск по серийному номеру</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leNumModul">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>261</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Модули для проверки</string>
     </property>
    </widget>
    <widget class="QListWidget" name="lwModul">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>751</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbCheck">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>320</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Исправен</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbBroken">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>320</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Неисправен</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>353</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Проверенные модули</string>
     </property>
    </widget>
    <widget class="QListWidget" name="lwChecked">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>370</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="tbDelCheck">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>356</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>353</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Забракованные модули</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbDelBroken">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>356</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QListWidget" name="lwBroken">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>380</y>
       <width>370</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QRadioButton" name="rbNewDevice">
   <property name="geometry">
    <rect>
     <x>792</x>
     <y>50</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Новые</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbOldDevice">
   <property name="geometry">
    <rect>
     <x>792</x>
     <y>80</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Неисправные</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>OTKControlWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>719</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbOK</sender>
   <signal>clicked()</signal>
   <receiver>OTKControlWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>609</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
